aws_backendless_chef_ha.yml

```diff
@@ -0,0 +1,1052 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: >
+  AWS HA Chef Server 2.0 - Backendless Chef - Ubuntu Xenial Based
+  "Backendless" Chef utilizing AWS as Backend for PostgreSQL and Elasticsearch
+  Sets up the following:
+    Backendless Items
+    * Elasticsearch 2.3 Cluster
+    * PostgreSQL 9.5.4
+
+Parameters:
+
+  ## Domain Configuration
+  HostedZone:
+    Type: String
+    Default: domain.com
+    Description: must match a route53 hosted domain/zone
+
+  SSLCertificateARN:
+    Type: String
+    Default: 'arn:aws:iam::'
+    Description: SSL Certficate ARN for SSL Certficate
+
+  ## Re-Deploy Configuration
+  BackupFilename:
+    Type: String
+    Default: ''
+    Description: Enter the backup file to restore (e.g.; backup_06_Dec_16.tar)
+
+  ExistingBucketName:
+    Type: String
+    Default: ''
+    Description: Enter a Existing Bucket Name to to use (Leave blank to create new)
+
+  DBSnapShot:
+    Type: String
+    Default: ''
+    Description: Enter ARN of DB Snapshot to Create Database From (Leave blank to create a fresh database)
+
+  ## Chef Configuration
+  ChefSubdomain:
+    Type: String
+    Default: chef-test
+    AllowedValues:
+      - chef-a
+      - chef-b
+      - chef-test
+    Description: subdomain/prefix for chose hosted zone used for staging
```

Comment:
If only chef-a, chef-b and chef-test are allowed, may want to note that a/b are two sides to an expected blue-green switch out (maybe an explanation in the README.md if it is not already there).